src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -401,6 +409,37 @@ public void testSnapshotBuilder() throws Exception {
     List<String> searchConceptNames =
         searchConceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
     assertThat("expected concepts are returned", searchConceptNames, contains("concept1"));
+
+    var rollupCountsResponse =
+        dataRepoFixtures.getRollupCounts(
+            steward,
+            datasetId,
+            new SnapshotBuilderCountRequest()
```

Comment:
Maybe just a style preference, but I think it could be nice to not define the CountRequest inline and instead make it a constant

Reply:
I can make that change - I personally prefer the inlines, but I think we're moving more towards intermediate variables as a team, so I can adapt that way.